ush/generate_FV3LAM_wflow.sh

```diff
@@ -862,10 +871,11 @@ if [ "${DO_DACYCLE}" = "TRUE" ]; then
      'nggps_ic'   : false,
      'mountain'  : true,
      'warm_start' : true,
-   }
- 'gfs_physics_nml': {
-    'fh_dfi_radar': [${FH_DFI_RADAR[@]}],
-  }"
+   }"
+# commnet out for using current develop branch that has no radar tten code yet.
+# 'gfs_physics_nml': {
+#    'fh_dfi_radar': [${FH_DFI_RADAR[@]}],
+#  }"
```

Comment:
A question for this: 
So FH_DFI_RADAR in config.sh will not have effect since this is commented out?